graf2d/gpad/src/TPad.cxx

```diff
@@ -5309,6 +5312,16 @@ void TPad::RedrawAxis(Option_t *option)
       else                   hobj->DrawCopy("sameaxis");
    }
 
+   if (opt.Contains("f")) {
+      auto b = new TBox(gPad->GetUxmin(), gPad->GetUymin(),
+                        gPad->GetUxmax(), gPad->GetUymax());
+      b->SetFillStyle(0);
+      b->SetLineStyle(1);
```

Comment:
So the fill and line styles are hard-coded?

Reply:
Yes, it is to redraw the line frame around a plot when some drawing just at the edge erased it (to be used in seldom cases but there was a request and it makes sense).  RedrawAxis is the place to put that. What we want to keep are the Line Width and the Line Color only.

Reply:
But yes in principal we cans that this two values from the frame ... let me try

Reply:
Well only the line style .. the fill style must be hollow (to redraw the line border only)

Reply:
OK, you know better than me...